config.yaml

```diff
@@ -69,12 +69,15 @@ directories:
 
 # Plotting scheduling parameters
 scheduling:
-        # Don't run a job on a particular temp dir until all existing jobs
-        # have progresed at least this far.  Phase major corresponds to the
-        # plot phase, phase minor corresponds to the table or table pair
-        # in sequence.
+        # Run a job on a particular temp dir only if the number of existing jobs
+        # before tmpdir_stagger_phase_major tmpdir_stagger_phase_minor
+        # is less than tmpdir_stagger_phase_limit.
+        # Phase major corresponds to the plot phase, phase minor corresponds to
+        # the table or table pair in sequence, phase limit corresponds to
+        # the number of plots allowed before [phase major, phase minor]
         tmpdir_stagger_phase_major: 2
         tmpdir_stagger_phase_minor: 1
+        tmpdir_stagger_phase_limit: 2
```

Comment:
I guess a value of '2' in the example config is more illustrative of how one might use this, but I expect '1' to be the more common setting.  Should we make '1' the default in the example config?

Reply:
Right, 1 makes more sense as the default. I'll go ahead and update it.